recipes/replit/meta.yaml

```diff
@@ -0,0 +1,54 @@
+{% set name = "replit" %}
+{% set version = "3.1.0" %}
+
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/replit-{{ version }}.tar.gz
+  sha256: fcea2df361a195ab6a6739ddba429227f36944075f8852e35f77a4031d263ab0
+  patches:
+    - 0001-add-script.patch
+
+build:
+  number: 0
+  noarch: python
+  script: {{ PYTHON }} -m pip install . -vv
+  entry_points:
+    - replit=replit.__main__:cli
+
+requirements:
+  host:
+    - pip
+    - poetry
```

Comment:
gah, probably just need `poetry-core`, but upstream asks for the full enchilada, which can bring a lot of unexpected things...

Reply:
So I should leave it?

Reply:
yeah, it's what they build with, so would leave it.